### ISSUE TITLE
CCv0 | version: static checks failing due to comment formattting

### ISSUE THREAD

**Issue by stevenhorsman** [kata-containers/kata-containers#5928]
When I ran it locally, the static checks failed (even on CCv0) with:

```
INFO: Running 'static_check_versions' function
versions.yaml
  298:8     warning  missing starting space in comment  (comments)
  299:8     warning  missing starting space in comment  (comments)
```

The offending lines are:
https://github.com/kata-containers/kata-containers/blob/1ee684021d0a53f0f0257a16c823ef047534b148/versions.yaml#L298-L301
(from PR #5893)

After manually adding the missing spaces, the static checks pass.

_Originally posted by @dubek in https://github.com/kata-containers/kata-containers/issues/5927#issuecomment-1356776885_

**Comment by stevenhorsman** [comment]
I'm not sure why the static check run on https://github.com/kata-containers/kata-containers/pull/5893 didn't find this, but I'll put a fix in to unblock things.

**Comment by stevenhorsman** [comment]
Closed with #5929 merging
